core/unix/loader.c

```diff
@@ -1856,6 +1856,7 @@ privload_early_inject(void **sp, byte *old_libdr_base, size_t old_libdr_size)
      * have to tell get_application_name() to use this path.
      */
     set_executable_path(exe_path);
+    set_application_cl_args((int *) argc, argv);
```

Comment:
This is for early injection -- what about late injection?  If we punt on that for now we should document it as `-late` is still a documented option to `drrun`.

Reply:
Done.